tests/ops/api/v1/endpoints/test_config_endpoints.py

```diff
@@ -330,23 +330,22 @@ def test_patch_application_config_updates_cors_domains_in_middleware(
 
         assert set(current_cors_middleware.options["allow_origins"]) == set(
             payload["security"]["cors_origins"]
-        )
+        ).union(set(original_cors_middleware_origins))
 
         # now ensure that the middleware update was effective by trying
         # some sample requests with different origins
 
-        # first, try with an original allowed origin, which should no longer be accepted
+        # first, try with an original allowed origin, which should still be accepted
```

Comment:
😝

Reply:
this diff would probably suffice as the entire PR description! 😆